pkgs/by-name/ve/vesktop/package.nix

```diff
@@ -128,6 +129,7 @@ stdenv.mkDerivation (finalAttrs: {
       makeWrapper ${electron}/bin/electron $out/bin/vesktop \
         --add-flags $out/opt/Vesktop/resources/app.asar \
         ${lib.optionalString withTTS "--add-flags \"--enable-speech-dispatcher\""} \
+        ${lib.optionalString withWaylandIME "--add-flags \"--enable-wayland-ime\""} \
         --add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--ozone-platform-hint=auto --enable-features=WaylandWindowDecorations}}"
```

Comment:
Maybe add it here? Does `--enable-wayland-ime` bring any overhead to people not using an IME?

Reply:
The value is set to false by default, so users who do not want it should not get any extra overhead.

Reply:
I moved that flag to `NIXOS_OZONE_WL` and removed the option I created.